crates/core/src/subscription/query.rs

```diff
@@ -52,6 +52,8 @@ pub fn compile_read_only_query(
                 return Err(SubscriptionError::SideEffect(Crud::Create(DbType::Table)).into())
             }
             CrudExpr::Drop { kind, .. } => return Err(SubscriptionError::SideEffect(Crud::Drop(kind)).into()),
+            CrudExpr::SetConfig { .. } => return Err(SubscriptionError::SideEffect(Crud::Config).into()),
+            CrudExpr::ReadConfig { .. } => return Err(SubscriptionError::SideEffect(Crud::Config).into()),
```

Comment:
Should we perhaps call these `SetVar` and `ReadVar`? Or `SetParam` and `ReadParam`?